src/pudl/transform/epacems.py

```diff
@@ -191,11 +239,14 @@ def correct_gross_load_mw(df: pd.DataFrame) -> pd.DataFrame:
     # This is rare, so don't bother most of the time.
     bad = df["gross_load_mw"] > 2000
     if bad.any():
-        df.loc[bad, "gross_load_mw"] = df.loc[bad, "gross_load_mw"] / 1000
+        df.loc[bad, "gross_load_mw"] = df.gross_load_mw / 1000
     return df
```

Comment:
I updated this line because I thought it was incorrect! But I'd love someone to verify.

Reply:
i believe either are technically correct. i tend to think the previous version is more clear, but your version is more concise. the `loc` on both sides of the `=` indicates that your only trying to update/calculate a portion of the `df`. Your edit would work but you have different indexes on both sides of this equation (the column masked w/ `bad` on one side and the full column / 100 on the other. `loc` is smart enough to check the indexes and only apply the `/ 1000` version to the masked columns. I'm not sure which pattern is cleaner tbh.